examples/migraphx/custom_op_hip_kernel/CMakeLists.txt

```diff
@@ -20,18 +20,62 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
-#####################################################################################
+# ####################################################################################
+
+function(rocm_local_targets_migx VARIABLE)
+    set(${VARIABLE} "NOTFOUND" PARENT_SCOPE)
+    find_program(_rocm_agent_enumerator rocm_agent_enumerator HINTS /opt/rocm/bin ENV ROCM_PATH)
+
+    if(NOT _rocm_agent_enumerator STREQUAL "_rocm_agent_enumerator-NOTFOUND")
+        execute_process(
+            COMMAND "${_rocm_agent_enumerator}"
+            RESULT_VARIABLE _found_agents
+            OUTPUT_VARIABLE _rocm_agents
+            ERROR_QUIET
+        )
+
+        if(_found_agents EQUAL 0)
+            cmake_policy(SET CMP0007 NEW)
+            string(REPLACE "\n" ";" _rocm_agents "${_rocm_agents}")
+            list(POP_BACK _rocm_agents)
+            list(REMOVE_DUPLICATES _rocm_agents)
+            unset(result)
+
+            foreach(agent IN LISTS _rocm_agents)
+                if(NOT agent STREQUAL "gfx000")
+                    list(APPEND result "${agent}")
+                endif()
+            endforeach()
+
+            if(result)
+                set(${VARIABLE} "${result}" PARENT_SCOPE)
+            endif()
+        endif()
+    endif()
+endfunction()
```

Comment:
Workarounds like this should not go into examples. Users will copy and use this after it no longer is needed.

Reply:
Okay, for this example then we can just pass `-DAMDGPU_TARGET` manually then ?

Reply:
> Okay, for this example then we can just pass -DAMDGPU_TARGET manually then ?

You would pass `-DGPU_TARGETS=...`.

Reply:
Okay. Modified README.md to reflect that.